src/System.Management.Automation/engine/Modules/ImportModuleCommand.cs

```diff
@@ -820,6 +841,23 @@ private PSModuleInfo ImportModule_LocallyViaName(ImportModuleOptions importModul
             return null;
         }
 
+        private PSModuleInfo ImportModule_LocallyViaFQName(ImportModuleOptions importModuleOptions, ModuleSpecification modulespec)
+        {
+            RequiredVersion = modulespec.RequiredVersion;
+            MinimumVersion = modulespec.Version;
+            MaximumVersion = modulespec.MaximumVersion;
+            BaseGuid = modulespec.Guid;
+
+            PSModuleInfo foundModule = ImportModule_LocallyViaName(importModuleOptions, modulespec.Name);
+            ApplicationInsightsTelemetry.SendTelemetryMetric(TelemetryType.ModuleLoad, foundModule.Name);
```

Comment:
Couldn't foundModule be null here?

Reply:
Moved it into the following `if (foundModule != null)`.